src/EnergyPlus/WindowEquivalentLayer.cc

```diff
@@ -5489,6 +5496,9 @@ namespace WindowEquivalentLayer {
         // New code follows from here - for calculating Ucg and SHGC
         // NOTE: This code can be bypassed if
         //       indices of merit are not needed
+        if (!(present(UCG) && present(SHGC))) {
```

Comment:
I'm not sure how `IM_ON` got set before, but can you do it without requiring optionals?

Reply:
Let me think If I can avoid requiring optional variables.

Reply:
Removed the optional arguments.